src/main/webapp/app/quiz/participate/quiz-scoring-infostudent-modal/quiz-scoring-infostudent-modal.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+    selector: 'jhi-quiz-scoring-infostudent-modal',
+    templateUrl: './quiz-scoring-infostudent-modal.component.html',
+    styles: []
+})
+export class QuizScoringInfostudentModalComponent implements OnInit {
+    constructor(private modalService: NgbModal) {}
+
+    @Input() score: number;
+    @Input() question: number;
+    @Input() scoringType: string;
```

Comment:
instead of scoringType and questionType (below), I would pass the actual question, because it already has an attribute type and scoringType.
In general we should try to reduce the number of atomic inputs (i.e. number, boolean, string) and rather have more complex object types as inputs (e.g. question)

Reply:
Reduced the 8 imports to 5 imports